### PR TITLE
feat: update dependencies to latest

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: examples/verify.rs [hunk 1]
```diff
@@ -46,11 +47,14 @@ macro_rules! measure {
 fn run(num_messages: usize) {
     println!("dancing with {} messages", num_messages);
 
-    let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+    let mut rng = XorShiftRng::from_seed([
+        0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06, 0xbc,
```
Comment (vmx) [inline examples/verify.rs]: Is it intentional that the forth byte changed from 0x3d to 0x5d?

File: src/key.rs [hunk 2]
```diff
@@ -135,11 +133,15 @@ impl Serialize for PublicKey {
...
 
-    use rand::{SeedableRng, XorShiftRng};
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
 
     #[test]
     fn test_bytes_roundtrip() {
-        let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+        let rng = &mut XorShiftRng::from_seed([
+            0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06,
```
Comment (vmx) [inline src/key.rs]: Is it intentional that the forth byte changed from 0x3d to 0x5d? At least also the verification was changed accordingly.

File: src/signature.rs [hunk 3]
```diff
@@ -94,17 +95,21 @@ pub fn verify(signature: &Signature, hashes: &[G2], public_keys: &[PublicKey]) -
...
 
-    use rand::{Rng, SeedableRng, XorShiftRng};
+    use rand::{Rng, SeedableRng};
+    use rand_xorshift::XorShiftRng;
 
     #[test]
     fn basic_aggregation() {
-        let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+        let mut rng = XorShiftRng::from_seed([
+            0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06,
```
Comment (vmx) [inline src/signature.rs]: Same as above.

File: src/signature.rs [hunk 4]
```diff
@@ -138,8 +143,11 @@ mod tests {
 
     #[test]
     fn test_bytes_roundtrip() {
-        let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
-        let sk = PrivateKey::generate(rng);
+        let mut rng = XorShiftRng::from_seed([
+            0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06,
```
Comment (vmx) [inline src/signature.rs]: Same as above.

File: examples/verify.rs (hunk 1 above)
Comment (dignifiedquire) [reply examples/verify.rs]: just adding some bytes, not particularly important as it is just the test seed

**Pull request merged**